projects/optic/src/commands/oas/operations/infer-path-structure.ts

```diff
@@ -395,7 +395,7 @@ function reducePathPattern(component: PathComponentCandidate): string {
 
 function fragmentize(path: string): string[] {
   if (path.length === 0 || !path.startsWith('/')) {
-    throw new Error(`Malformed path '${path}'`);
```

Comment:
I'm not sure why this was throwing an error - a base path is a valid path, and should be treated as an empty array (similarly with other parts of the code)

Reply:
Yes this looks correct. I'm not 100% sure how it'd get there, but I don't think this code handles the case where you get something like "api/v4" -- where "api" doesn't have the required leading "/".

Reply:
I see - I updated this to have a check like:
```
if (!path.startsWith('/')) {
  path = `/${path};`
}
```

this will return `[]` for `''`, `'/'` and also correctly handle `api/v4` -> fragmentizing it correctly (which i'm not sure how it'd be set that way since all of this is coming from hars)